### ISSUE TITLE
El Capitan 10.11 Performance Issues

### ISSUE THREAD

**Issue by ChrisChiera** [adobe-photoshop/spaces-design#1585]
Currently when using in El Capitan OS X 10.11 (15A178w) on a 5K iMace with 32GB ram, the beach ball constantly appears and momentarily freezes almost every 4 or so seconds as you attempt to perform any action.

**Comment by jsbache** [comment]
Thanks for the report.
We have just started looking at OSX 10.11. You may also notice a redraw artifact around the application frame when you toggle in and out of Design Space.

**Comment by chadrolfs** [comment]
https://watsonexp.corp.adobe.com/#bug=4010180

**Comment by iwehrman** [comment]
A friendly twitter user also reports that Smart Guide labels and tooltips aren't showing on El Capitan:https://twitter.com/bbaiIey/status/613380410521718784

**Comment by baiIey** [comment]
Thanks @iwehrman. This is what I'm seeing in PS CC (2015) on OS X El Capitan 10.11 beta 2. 

Feel free to let me know if you have any questions or comments.
![ps-el-capitan-guides](https://cloud.githubusercontent.com/assets/701984/8313403/4e2801c8-1997-11e5-8fba-c9f1d0cd9a55.gif)

**Comment by wuyue1990** [comment]
same issue here. should be caused by the new font.

**Comment by tiburbage** [comment]
FYI I have closed the Watson bug https://watsonexp.corp.adobe.com//#bug=4010180 based on the description of the problem reported in the Watson bug report, and the hardware/OS context. I was able to test our current builds on an iMac 5K with the 10.11 Beta 6 build and was using Design Space fully maximized and with our VermilionArtboards.psd test doc. The user experience was not significantly different from that with current builds on a MacBook Pro running 10.10 or 10.11 Beta 6. We will continue to test with Beta 6 and subsequent pre-release builds looking for deltas between 10.10 and10.11 of course.
